utils/dataset_validator.py

```diff
@@ -0,0 +1,109 @@
+from flask_uuid import UUID_RE
+
+DATASET_NAME_LEN_MIN = 1
+DATASET_NAME_LEN_MAX = 100
+
+MIN_CLASSES = 2
+CLASS_NAME_LEN_MIN = 1
+CLASS_NAME_LEN_MAX = 100
+
+MIN_RECORDINGS_IN_CLASS = 2
+
+
+def validate(dataset, complete=False):
```

Comment:
I added a docsting with some info. Datasets are not always created by the app. This is also used to validate datasets that are submitted to the "API" endpoint. It is used by our creation UI, but it's still important to validate.